packages/coinstac-ui/app/render/components/app.js

```diff
@@ -1,29 +1,32 @@
 import Notify from './notification';
-import React, { PropTypes } from 'react';
+import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
 
-function App(props) {
-  const isLoading = props.loading.isLoading;
-  return (
-    <div className="app">
-      <ul id="spinner" className={isLoading ? 'is-loading' : ''}>
-        <li></li>
-        <li></li>
-        <li></li>
-        <li></li>
-      </ul>
-
-      {props.children}
-
-      <ul id="loading_list">
-        {Object.keys(props.loading.wip).map((loadingKey, ndx) => {
-          return (<li key={ndx}>{loadingKey}</li>);
-        })}
-      </ul>
-
-      <Notify />
-    </div>
-  );
+class App extends Component { // eslint-disable-line react/prefer-stateless-function
```

Comment:
There’s some bug with _react-hot-loader_ not liking functional components. We can revert when that plugin hits v3.